File: tests/redisTest.py
#import modules
import redis

r = redis.Redis(host="192.168.2.8", port=6379, db=0, password="password")
r.set("openTrades", "4")
r.set("simulatedSum", "0.12")
r.set("sumResult", "0.01")
hello = "yeye" + r.get('openTrades').decode('utf-8')
print(hello)

File: tests/redisTest.js
// import modules
const redis = require("redis");

// load redis connection vars from environment
const redisClient = redis.createClient({
	    host: "192.168.2.8",
	    password: "password"
});

redisClient.set('simulatedWinner', 200000);

redisClient.get('recentSumResult', (err, reply) => {
	if (err) throw err;
	console.log(reply);
});
